tests/tests/swfs/avm2/displayobject_opaque_background/test.toml

```diff
@@ -0,0 +1,7 @@
+num_frames = 1
+
+#[image_comparison]
```

Comment:
You can leave these uncommented but add `ignore = true`, so we can easily see that this test exists but doesn't work yet